addons/survey/models/survey_survey.py

```diff
@@ -1100,6 +1100,13 @@ def action_end_session(self):
         self.user_input_ids.sudo().write({'state': 'done'})
         self.env['bus.bus']._sendone(self.access_token, 'end_session', {})
 
+    def action_assign_yourself(self):
+        """Assign current user to a survey without responsible"""
+        self.ensure_one()
+        if not self.user_ids:
+            self.sudo().write({'user_ids': self.env.user})
```

Comment:
shouldn't we show an error if someone is already assigned ?
or maybe we should hide the button ?